core/src/test/java/io/grpc/internal/ManagedChannelImplTest.java

```diff
@@ -2211,4 +2286,17 @@ private static ChannelStats getStats(
       Instrumented<ChannelStats> instrumented) throws Exception {
     return instrumented.getStats().get();
   }
+
+  private FakeClock.ScheduledTask getNameResolverBackoff() {
+    FakeClock.ScheduledTask nameResolverBackoff = null;
+    for (FakeClock.ScheduledTask task : timer.getPendingTasks()) {
+      if (task.command.toString().contains("NameResolverBackoff")) {
```

Comment:
You may consider using `getPendingTasks(TaskFilter filter)` instead.

Reply:
Done